### PR TITLE
deviation & zero rating fields optional and 10 >= contact_length <= 30

### PR DESCRIPTION
**Pull request by lubegasimon**

### CODE REVIEW

File: src/views/components/EvaluationForm/index.tsx [hunk 1]
```diff
@@ -333,7 +326,7 @@ export default class EvaluationForm extends React.Component<Props, State> {
               label="Phone Number"
               name="customerTel"
               minLength={10}
-              maxLength={10}
+              maxLength={30}
               pattern="[0]{1}[0-9]{9}"
```
Comment (epicallan) [inline src/views/components/EvaluationForm/index.tsx]: you should as well get rid of the `pattern` attribute such that the onus is on them to input correctly formatted phone numbers.

File: src/views/components/NPSdataTable/index.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ export interface NPS extends TableData {
   rating: number;
   ratingReason: string;
   crmCaptureCorrect: boolean;
-  crmCaptureReason: string;
+  crmCapturedReason: string;
```
Comment (epicallan) [inline src/views/components/NPSdataTable/index.tsx]: why this change ? I don't think it's required.

File: src/views/components/EvaluationDataTable/index.stories.tsx [hunk 3]
```diff
@@ -44,7 +44,7 @@ export const evalTestData = [
       evaluator: "simon",
       agentName: "thanos",
       comment: "failed to apologize for failure to",
-      customerTel: "10",
+      customerTel: "30",
```
Comment (epicallan) [inline src/views/components/EvaluationDataTable/index.stories.tsx]: These changes are not actually required.

File: src/views/components/EvaluationDataTable/index.stories.tsx [hunk 4]
```diff
@@ -76,7 +76,7 @@ export const evalTestData = [
     evalAttrs: {
       date: "2019-04-17",
       evaluator: "aleku399",
-      customerTel: "10",
+      customerTel: "30",
```
Comment (epicallan) [inline src/views/components/EvaluationDataTable/index.stories.tsx]: This change is not necessary.

File: src/views/components/NPSdataTable/index.tsx (hunk 2 above)
Comment (lubegasimon) [reply src/views/components/NPSdataTable/index.tsx]: storybook had ```crmCapturedReason```, so it could open because they were incompatible .

File: src/views/components/EvaluationDataTable/index.stories.tsx (hunk 3 above)
Comment (lubegasimon) [reply src/views/components/EvaluationDataTable/index.stories.tsx]: yea, so as to open storybook for console logging purposes, they're needed

File: src/views/components/EvaluationDataTable/index.stories.tsx (hunk 4 above)
Comment (lubegasimon) [reply src/views/components/EvaluationDataTable/index.stories.tsx]: yea, but so as to open storybook for console logging purposes, they're needed

File: src/views/components/NPSdataTable/index.tsx (hunk 2 above)
Comment (epicallan) [reply src/views/components/NPSdataTable/index.tsx]: it should actually be `crmCaptureReason` in storybook to. The source of truth should be the type in the react component module

File: src/views/components/EvaluationDataTable/index.stories.tsx (hunk 3 above)
Comment (epicallan) [reply src/views/components/EvaluationDataTable/index.stories.tsx]: BTW you should realise the `cutomerTel` should be a value of either `256771849086` or `0771849086`. I would suggest to change it to search a value if you feel the need to change it

File: src/views/components/EvaluationDataTable/index.stories.tsx (hunk 4 above)
Comment (epicallan) [reply src/views/components/EvaluationDataTable/index.stories.tsx]: see comment above

File: src/views/components/EvaluationDataTable/index.stories.tsx (hunk 3 above)
Comment (lubegasimon) [reply src/views/components/EvaluationDataTable/index.stories.tsx]: Oh I get, I did random changes..

File: src/views/components/NPSdataTable/index.tsx (hunk 2 above)
Comment (epicallan) [reply src/views/components/NPSdataTable/index.tsx]: Please address this and change it back @lubegasimon

**Comment by lubegasimon** [comment]
I did please, checkout https://github.com/freelyformd/eval-cso/pull/195/commits/dbd6b3f5bc32793c7f63f224f757627ac12109c6

**Comment by epicallan** [comment]
> I did please, checkout

If you had then it wouldn't show up in this PR, but it does!

**Comment by epicallan** [comment]
You should fix it in this PR too, we shouldn't merge code with glaring mistakes.

**Comment by lubegasimon** [comment]
> > I did please, checkout
> 
> If you had then it wouldn't show up in this PR, but it does!

this was before the PR I resolved it from..
